lib/init.coffee

```diff
@@ -103,7 +103,41 @@ module.exports =
         reResult = re.exec(toParse)
       ret
 
+    # only elixir 1.3+
     parseWarning = (toParse, textEditor) ->
+      console.log('to parse', toParse)
+
+      ret = []
+      re = ///
+        warning:\ (.*)\n # warning
+        \ \ (.*):([0-9]+) # file and file number
+        ///g
+      reResult = re.exec(toParse)
+
+      while reResult?
+        console.log(reResult)
```

Comment:
This should be wrapped in a check on `atom.inDevMode()`, if not removed entirely.